setup.py

```diff
@@ -7,6 +7,22 @@
 
 ROOT = Path(__file__).parent
 
+# Note: In GluonTS we use git tags to manage versions. A new release is created
+# by creating a new tag on GitHub through their release mechanism. Thus,
+# `gluonts.__version__` uses the latest available git tag. If there are
+# additional commits on top of the tagged commit, we extend the version
+# information and append a `.dev0+g{commit_id}` to the version. If there are
+# uncommitted changes, an additional `.dirty` is appended to the version.
+# Since we always rely on the latest available tag, it is important to ensure
+# that the latest tag in the `main` branch is `v0` and not a more specific
+# version like `v0.x`, since the `main` branch should be independent from a
+# more specific version. This means that we can't tag a commit on `main` when
```

Comment:
`main` or `dev`?

Reply:
Good point, we are using `dev` of course, thanks!